docs/user_guide.md

```diff
@@ -58,9 +58,19 @@ The following screenshot show the "Settings" menu, where you can set workspace f
 
 <img src="../images/user_guide/create_workspace/workspace_settings.png" srcset="../images/user_guide/create_workspace/workspace_settings.png 2x" alt="create_workspace/WorkspaceSettings.png"/>
 
+## Browsing existing open source Widgets on GitHub
```

Comment:
The recommended way of searching components (widgets, operators and mashups) MUST be the marketplace support, so this section should be moved later and be the alternative.

Reply:
The reason for putting the _look-on-github_ bit first was to ensure that users download fresh sources. May be this can be reworded - something like _"getting latest bleeding edge development"_

Reply:
Fixed  cca64bbc4